auto_round/utils.py

```diff
@@ -583,37 +586,46 @@ def is_autoround_exllamav2_available():
     return res
 
 
-def get_autogptq_backend_config(backend, bits=4):
+def get_autogptq_infer_linear(backend, bits=4, group_size=128, sym=False):
     use_triton = False
     disable_exllamav2 = False
     disable_exllamav1 = False
     disable_marlin = True
     use_qigen = False
-    if backend == "gptq:qigen":
+    if "qigen" in backend:
         use_qigen = True
-    if backend == "gptq:triton":  ##TODO refine the code
+    elif "triton" in backend:
         use_triton = True
-    if backend == "gptq:marlin":
+    elif "marlin" in backend:
```

Comment:
asym is not supported for marlin in infer_linear situation, right?

Reply:
yes